### ISSUE TITLE
Unable to find dotnet when building with dotnet cli

### ISSUE THREAD

**Issue by Nikodage** [oleg-shilo/wixsharp#1463]
Hello,

I am getting a NullException in Compiler.Run when EnsureDtfTool is called. ExternalTool.Locate(exe) will return null with exe being "dotnet". Changing the code in EnsureDtfTool to look for "dotnet.exe" instead of "dotnet" fixes it. 
Is that an expected behavior? Thank you

PS: This library is incredible. Thank you for providing it

**Comment by oleg-shilo** [comment]
> This library is incredible. Thank you for providing it

Thank you. Glad you find it useful :)

>"dotnet.exe" instead of "dotnet" fixes it.

I changed the code. All good. But it's strange. Does it mean that if ou execute in cmd-prompt `dotnet` it will not work unless it's `dotnet.exe`?

**Comment by oleg-shilo** [comment]
Yep, this is what I was suspecting. Somehow your dotnet setup did not go as expected. Or you have a tricky OS config, preventing execution of any app without a full file name (try `notepad`).
But the workaround you suggested is OK to keep. At least it will help people with the same problem as yours.
